File: src/main/java/com/learnershi/rclasssocket/entity/SpeedQuizSubmit.java
package com.learnershi.rclasssocket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

/**
 * SpeedQuizSubmit
 *
 * @author GyeongSik
 */
@Accessors(chain = true)
@NoArgsConstructor
@Getter
@Setter
@Document(collection = "speedQuizSubmit")
public class SpeedQuizSubmit extends ClassRoomEntity {
    private String userName;
    private List testResult;
    private String correctCount;
    private String incorrectCount;
}

File: src/main/java/com/learnershi/rclasssocket/entity/SpeedQuizSubmit.kt
package com.learnershi.rclasssocket.entity

import org.springframework.data.mongodb.core.mapping.Document

/**
 * SpeedQuizSubmit
 *
 */
@Document(collection = "speedQuizSubmit")
class SpeedQuizSubmit : ClassRoomEntity() {
    private val userName: String? = null
    private val testResult: List<*>? = null
    private val correctCount: String? = null
    private val incorrectCount: String? = null
}